exchanges/kucoin/kucoin.go

```diff
@@ -2448,7 +2464,7 @@ func (ku *Kucoin) SendAuthHTTPRequest(ctx context.Context, ePath exchange.URL, e
 			"KC-API-SIGN":        crypto.Base64Encode(signHash),
 			"KC-API-TIMESTAMP":   timeStamp,
 			"KC-API-PASSPHRASE":  crypto.Base64Encode(passPhraseHash),
```

Comment:
Hello, I'm currently getting failures with the passphrase. 
```https://api.kucoin.com/api/v1/margin/account```
``` Kucoin unsuccessful HTTP status code: 401 raw response: {"code":"400004","msg":"Invalid KC-API-PASSPHRASE"}, ```


I believe its because of this:
https://www.kucoin.com/docs/basic-info/connection-method/authentication/signing-a-message
>    For API key-V1.0, please pass requests in plaintext.
>    For API key-V2.0, please Specify KC-API-KEY-VERSION as 2 --> Encrypt passphrase with HMAC-sha256 via API-Secret --> Encode contents by base64 before you pass the request.
    For API key-V3.0, please Specify KC-API-KEY-VERSION as 3 --> Encrypt passphrase with HMAC-sha256 via API-Secret --> Encode contents by base64 before you pass the request.